content/docs/command-reference/plots/index.md

```diff
@@ -280,7 +250,157 @@ file:///Users/usr/src/dvc_plots/index.html
 ![](/img/plots_show_confusion.svg)
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
-> predefined in DVC (found in `.dvc/plots/confusion.json`).
+> predefined in DVC.
+
+We can use `confusion_normalized` template to normalize the results:
+
+```dvc
+$ dvc plots show classes.csv --template confusion_normalized -x actual -y predicted
+file:///Users/usr/src/dvc_plots/index.html
+```
+
+![](/img/plots_show_confusion_normalized.svg)
+
+## Example: simple plot definition
+
+Let's get back to the `logs.csv` data:
+
+```
+# logs.csv
+epoch,loss,accuracy
+1,0.19,0.81
+2,0.11,0.89
+3,0.07,0.93
+4,0.04,0.96
+```
+
+Minimal plot definition we can put in `dvc.yaml` is simply data source path
+relative to `dvc.yaml` file:
+
+```yaml
+# dvc.yaml
+stages:
+  train:
+    cmd: echo "Hello world"
+
+plots:
+  logs.csv:
+```
+
+```dvc
+$ dvc plots show
+file:///Users/usr/src/dvc_plots/index.html
+```
+
+![](/img/plots_show_spec_default.svg)
+
+We can customize it:
+
+```yaml
+# dvc.yaml
+stages:
+  train:
+    cmd: echo "Hello world"
+
+plots:
+  logs.csv:
+    x: epoch
+    y: accuracy
+    title: Displaying accuracy
+    x_label: This is epoch
+    y_label: This is accuracy
+```
+
+```dvc
+$ dvc plots show
+file:///Users/usr/src/dvc_plots/index.html
+```
+
+![](/img/plots_show_spec_simple_custom.svg)
+
+## Example: multiple data-series plot definition:
+
+Data in `training_data.csv`:
+
+```csv
+epoch,train_loss,test_loss
+1,0.33,0.4
+2,0.3,0.28
+3,0.2,0.25
+4,0.1,0.23
+```
+
+```yaml
+# dvc.yaml
+stages:
+  train:
+    cmd: echo "Hello world"
+
+plots:
+  test_vs_train_loss:
+    x: epoch
+    y:
+      training_data.csv: [test_loss, train_loss]
+    title: Compare loss training versus test
+```
+
+![](/img/plots_show_spec_multiple_columns.svg)
+
+## Example: sourcing data from different files
+
+Lets prepare comparison for confusion matrix data between test set and training
+set:
+
+```csv
+# train_classes.csv
+actual_class,predicted_class
+dog,dog
+dog,dog
+dog,dog
+dog,bird
+cat,cat
+cat,cat
+cat,cat
+cat,dog
+bird,bird
+bird,bird
+bird,bird
+bird,dog
+```
+
+```csv
+# test_classes.csv
+actual_class,predicted_class
+dog,dog
+dog,dog
+dog,cat
+bird,bird
+bird,bird
+bird,cat
+cat,cat
+cat,cat
+cat,bird
+```
+
+```yaml
+# dvc.yaml
+stages:
+  train:
+    cmd: echo "Hello world"
```

Comment:
make it more realistic? or remove completely (we don't anything like this in all the other cases)

Reply:
it makes reproducible though ...

make it "Training a model ... " at least?

Reply:
I think in ref examples realistic > reproducible. We don't expect readers to run the examples as in a tutorial right? They're mostly to see them, maybe copy the code but change it to fit their project.

Reply:
Modified.